### PR TITLE
components added: alert and button

### PR DESCRIPTION
**Pull request by partha8**
Created alert and button components. The links are active for merged components. I will add the description of each component later. Thank you for reviewing.

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  border-radius: var(--borderRadius);
+}
+
+.alert p {
+  margin: 0;
+}
+.alert-primary {
+  background: var(--clr-blue-primary);
+  color: var(--clr-white);
+}
```
Comment (ApurvaSawant11) [inline components/alert/alert.css:18]: Add one line space before defining the next class.  It helps in readability.

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me 👍🏽

File: components/alert/alert.css (hunk 1 above)
Comment (partha8) [reply components/alert/alert.css:18]: Thank you for reviewing, I will make the changes!

**Review by shahbazibnafzal — APPROVED** [review]
Looks good

**Comment by partha8** [comment]
Got two reviews from my peers and made the changes. Merging the branch.

**Pull request merged**